index.js

```diff
@@ -79,6 +124,8 @@ function processParams (opts) {
   }
 
   const supportedEncodings = ['gzip', 'deflate', 'identity']
+
+  /* istanbul ignore next */
```

Comment:
I think we can remove the if completely right? we have brotli everywhere?

Reply:
AFAIK no. fastify-compress declares support for Node >= 10.16, while brotli (according to docs) landed in 11.7.0. Am I wrong?

Reply:
It got backported in Node 10.16. See https://nodejs.org/docs/latest-v10.x/api/zlib.html

Reply:
Oh, nice. I'll remove from both compress and uncompress part then.

Reply:
Done